frontend/src/components/AddMaterialDialog.jsx

```diff
@@ -30,8 +30,8 @@ export default function AddMaterialDialog({ open, onClose, onSave }) {
 	useEffect(() => {
 		const fetchArtifactTypes = async () => {
 			try {
-				const response = await http.get("/artifactTypes");
-				setArtifactTypes(response.data); // Update state with fetched artifact types
+				//const response = await http.get("/artifactTypes");//TODO: figure out
```

Comment:
It takes a special kind of idiot to leave commented out code in a document. tf is this? If we don't need it remove it. jeezuz

Reply:
Created issue #33 to handle a discussion on how this should be handled in the future.